cypress/support/pages/article.pageObject.js

```diff
@@ -0,0 +1,29 @@
+import PageObject from '../PageObject';
+
+class ArticlePageObject extends PageObject {
+    url = '/#/articles';
+    
+    get editArticleBtn() {
+        return cy.getByDataCy('edit-article-btn');
+    }
+
+    get deleteArticleBtn() {
+        return cy.getByDataCy('delete-article-btn');
+    }
+
+    clickOnEditArticleBtn() {
+        this.editArticleBtn
+            .click();
+    }
+
+    clickOnDeleteArticleBtn() {
+        this.deleteArticleBtn
+            .click();
+    }
+
+    visitArticle(title){
+        cy.visit(`/${title}`);
```

Comment:
Ensure that the URL path in `visitArticle` is correct for visiting a specific article. The current implementation assumes a direct path using the title, which might not be accurate if the URL structure is different (e.g., if it includes an ID or slug).